### PR TITLE
Mobile navigation system

### PR DESCRIPTION
**Pull request by adguernier**
- [x] install react-navigation/native 
- [x] try to play with to understand how it works
- [x] make a navigation between home (games list) and game details. Game detail only show the id

### CODE REVIEW

File: mobileapp/components/home/Home.tsx [hunk 1]
```diff
@@ -1,11 +1,10 @@
-import { StyleSheet, Text, View } from "react-native";
+import { Button, StatusBar, StyleSheet, Text, View } from "react-native";
```
Comment (paulSanchezMarmelab) [inline mobileapp/components/home/Home.tsx:1]: Some unused imports here. We should both check linter configuration for that.

**Review by paulSanchezMarmelab — CHANGES_REQUESTED** [review]

**Comment by fzaninotto** [comment]
Needs rebase

File: mobileapp/components/home/GameList.tsx [hunk 2]
```diff
@@ -19,7 +19,9 @@ export const GameList = () => {
       {games && (
         <FlatList
           data={games}
-          renderItem={({ item }) => <GameListItem id={item.id} />}
+          renderItem={({ item }) => (
+            <GameListItem id={item.id} navigation={navigation} />
```
Comment (fzaninotto) [inline mobileapp/components/home/GameList.tsx:23]: I'm surprised that you need to pass down the `navigation` instance. Isn't there a hook to grab it from anywhere? Alternatively, use a React Context to make it available to all the descendants of the main App component.

**Review by fzaninotto — APPROVED** [review]

File: mobileapp/components/home/GameList.tsx (hunk 2 above)
Comment (adguernier) [reply mobileapp/components/home/GameList.tsx:23]: I simply followed the [doc](https://reactnative.dev/docs/navigation#usage). 

@paulSanchezMarmelab suggested me to use a context too, so i will do this

**Comment by fzaninotto** [comment]
Needs rebase

File: mobileapp/components/home/GameList.tsx (hunk 2 above)
Comment (fzaninotto) [reply mobileapp/components/home/GameList.tsx:22]: no, not urgent

**Pull request merged**